materials/databases_intro.md

```diff
@@ -5,81 +5,104 @@ title: Introduction to Databases
 
 ## Database management systems
 
-* We will be working with Relational Databases
-* We will use SQLite
-    * Simple, almost no work to set up, single file
-	* But there are lots of alternatives
-	* Access - commonly used, GUI
-	* PostgreSQL - fast/powerful, lots of users
+* We will be working with 'Relational Databases'
+    * The concepts of relational database querying are core to understanding 
+      how to do similar things using programming languages such as R or Python.
+* We will primarily use SQLite
+    * Simple to use, 
+    * Almost no work to set up
+    * Stored in single file
+    * But, there are lots of alternatives
+	  * Access - commonly used, GUI
+	  * PostgreSQL - fast/powerful, lots of users
 
-**Open SQLite w/Portal DB**
+> Open and preview SQLite with `portal_mammals.sqlite`
 
-## Key features of database management systems
+### Key features of database management systems
 
 * Data is separate from manipulations of the data
 * Tables - store the data
 * Queries - store questions about the data
     * If we update the data, the query asks the same question of the new data
 
-## Relational databases
-
-**Demo in SQLite**
+### Relational databases
 
 * Data is stored in tables
     * One table per type of data
-	* Tables can be linked together to combine information
+    * Tables can be linked together to combine information
 * Each row contains a single record
     * A single observation or data point
 * Each column contains a single attribute
-    * A single type of information
+    * A single field or type of information
 
-## Types
+### Data Types
 
 * Fields in databases have types that define the kind of data they contain
 * Each field/column can only have one type
 * We have to define the types in advance
 * Types include
     * Integer
-	* Text
-	* Decimal/Double
-* Types are highly configurable because space is limiting
+    * Text
+    * Decimal/Double/Real/Float
+* Types are highly configurable for when space is limiting
     * Maximum value of integers
-	* Maximum length of text
-	* How many values before and after the decimal place
+    * Maximum length of text
+    * How many values before and after the decimal place
 
-## Primary keys
+### Primary keys
 
 * Every table "needs" a column (or set of columns) that is unique across
   records/row
-* This is called the primary key
-* The easiest way to do this is to use an Integer that increments every time a
+* This is called the 'Primary Key'
+* The easiest way to do this is to use an 'Integer' that increments every time a
   new record is added
 * Many databases that you import will already have a field like this
 
+### Database Control
+
+There are many ways to add and modify data in a database management system.
+
+1\. Enter the data into another program (e.g., a spreadsheet) and import it to SQLite
+
+> Do [Exercise 1 - Data entry validation in Excel]({{ site.baseurl }}/exercises/Qaqc-data-entry-validation-in-excel-SQL/). 
```

Comment:
There were no notes on this in `materials/databases_intro.md` or `lectures/sql_intro.md`. There is a `materials/access-forms.md` that might include relevant info, but nothing specifically to Excel.